core/src/test/scala/better/files/FileSpec.scala

```diff
@@ -305,7 +302,6 @@ class FileSpec extends FlatSpec with BeforeAndAfterEach with Matchers {
   }
 
   it should "compute correct checksum for non-zero length string" in {
-    implicit val charset = StandardCharsets.UTF_8
```

Comment:
I added this because the default for `writeText` is `Charset.defaultCharset()`, which is system dependent. Otherwise, technically this test can fail on somebody else's machine if their default charset is something unusual.